core/pom.xml

```diff
@@ -463,6 +469,41 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>build-rust-library</id>
+            <build>
+                <plugins>
+                    <!--
+                    The plugin is used in the development environments, that require Rust
+                    code changes. The binary need not be placed in the platform-specific directory
+                    as it is expected to be NOT PRESENT in all other environments. E.g. those
+                    that do not target Rust library changes.
+
+                    The runtime libquestdbr library will be picked up from the source control. It will
+                    be deposited to the source control via CI job.
+                    -->
+                    <plugin>
+                        <groupId>org.questdb</groupId>
+                        <artifactId>rust-maven-plugin</artifactId>
+                        <version>1.2.0</version>
+                        <executions>
+                            <execution>
+                                <id>qdb-build</id>
```

Comment:
rename `qdb-build` -> `qdbr-build`, given you've changed the crate name.

Alternatiely, rename the crate to `qdb` which is probably a better name since it's more consistent with `qdb-ent`. The lib name can always be renamed in `[lib] name = ...` in `Cargo.toml`.